anvil/config.py

```diff
@@ -9,12 +9,35 @@
 from reportengine.configparser import ConfigError, element_of, explicit_node
 
 from anvil.core import TrainingOutput
+from anvil.train import (
+    adam,
+    adadelta,
+    stochastic_gradient_descent,
+    reduce_lr_on_plateau,
+)
 from anvil.models import RealNVP
 from anvil.geometry import Geometry2D
-from anvil.distributions import normal_distribution, phi_four_action
+from anvil.distributions import *
```

Comment:
I'd really rather avoid this, if you need that many things from one module then you can just import the whole module rather than polluting the namespace.

See my other comment regarding defining the options dictionary in distribution though

Reply:
yeah that's fair.